lab_1_keywords_tfidf/main.py

```diff
@@ -118,7 +189,26 @@ def calculate_expected_frequency(
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if doc_freqs and isinstance(doc_freqs, dict) and isinstance(corpus_freqs, dict):
```

Comment:
revert the condition and return None immediately

Reply:
Done